modules/pafIdSystem.md

```diff
@@ -0,0 +1,102 @@
+# Prebid Addreaability Framework (OneKey)
+
+Insert description here
+https://github.com/prebid/addressability-framework
+https://github.com/prebid/paf-mvp-implementation
+
+## PAF Registration
+
+insert info here
+
+## PAF Configuration
+
+The pafData module depends on paf-lib.js existing in the page.
+
+Compile the pafData module into your Prebid build.
+You will also want to add the pafRtdProvider module as well.
+
+`gulp build --modules=userId,pafIdSystem,rtdModule,pafRtdProvider,appnexusBidAdapter`
+
+There are no custom configuration parameters for PAF. The module
+will retrieve the PAF data from the page if available and pass the 
+information to bidders. Here is a configuration example:
+
+```javascript
+pbjs.setConfig({
```

Comment:
Question: I'm not very familar with prebid conf. Is it correct to assume that the paf id module will still be run even if no config is set ?

Reply:
the module has to be in the userIds array for it to be run even if its included in the build, although no configuration parameters are required